backend/users.py

```diff
@@ -23,7 +23,7 @@
 # ============================================================================
 PASSWORD_SECRET = os.environ.get("PASSWORD_SECRET", uuid.uuid4().hex)
 
-JWT_LIFETIME = int(os.environ.get("JWT_LIFETIME", 3600))
+JWT_TOKEN_LIFETIME = int(os.environ.get("JWT_TOKEN_LIFETIME", 60)) * 60
```

Comment:
It's a bit confusing that this gets multiplied by 60 here, maybe the env variable could be more explicit like:
```suggestion
JWT_TOKEN_LIFETIME = int(os.environ.get("JWT_TOKEN_LIFETIME_MINUTES", 60)) * 60
```

Reply:
Do you think this makes sense, or should just stick with seconds to be more consistent? either way works!